src/sidebars/search/search.less

```diff
@@ -74,6 +78,17 @@
   .compact-search {
     display: block;
   }
+  .compact-search.expanded {
+    icon {
+      display: block;
+      #close-button {
+        position: relative;
+        float: right;
+        top: -16px;
+        right: 7px;
```

Comment:
Maybe a `cursor: pointer;` so it gives a better indication it can be clicked? Looks good aside from that, though